dbt/include/spark/macros/adapters.sql

```diff
@@ -215,9 +215,7 @@
   {% for column_name in column_dict %}
     {% set constraints = column_dict[column_name]['constraints'] %}
     {% for constraint in constraints %}
-      {% if constraint.type != 'not_null' %}
-        {{ exceptions.warn('Invalid constraint for column ' ~ column_name ~ '. Only `not_null` is supported.') }}
-      {% else %}
+      {% if constraint.type == 'not_null' %}
```

Comment:
Does this make sense?  I'm not sure why this is the behavior now if none of the constraints are enforced?

Reply:
There's been talk in other PR's of logging warnings that certain constraints aren't enforced. 

If you've got those covered upstream, and I think you do, I endorse this.